src/core/dbus-execute.c

```diff
@@ -1909,6 +1924,21 @@ int bus_exec_context_set_transient_property(
                 return 1;
         }
 
+        if (streq(name, "ProtectControlGroups")) {
```

Comment:
there should also be a parser for ProtectControlGroupsEx here, i.e. in enum format